The job **`x86_64-apple-1`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/6497703011/job/17647392795) [(plain)](https://github.com/rust-lang-ci/rust/commit/dbe2d0ae39b0bbba4fe224c8d12f2a587a3f5e2a/checks/17647392795/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
SCCACHE_BUCKET=rust-lang-ci-sccache2
SCRIPT=./x.py --stage 2 test --skip tests/ui --skip tests/rustdoc --skip tests/run-make-fulldeps
SHELL=/bin/bash
SHLVL=3
SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.aCOpFWOKEX/Listeners
STATS_EXTP=https://provjobdsettingscdn.blob.core.windows.net/settings/provjobdsettings-0.5.154/provjobd.data
STATS_RDCL=true
STATS_TIS=mining
STATS_TRP=true
---
failures:

---- [assembly] tests/assembly/x86_64-array-pair-load-store-merge.rs stdout ----

error: verification with 'FileCheck' failed
status: exit status: 1
command: "/Users/runner/work/rust/rust/build/x86_64-apple-darwin/ci-llvm/bin/FileCheck" "--input-file" "/Users/runner/work/rust/rust/build/x86_64-apple-darwin/test/assembly/x86_64-array-pair-load-store-merge/x86_64-array-pair-load-store-merge.s" "/Users/runner/work/rust/rust/tests/assembly/x86_64-array-pair-load-store-merge.rs" "--allow-unused-prefixes" "--check-prefixes" "CHECK,NONMSVC" "--dump-input-context" "100"
--- stderr -------------------------------
/Users/runner/work/rust/rust/tests/assembly/x86_64-array-pair-load-store-merge.rs:14:16: error: CHECK-NOT: excluded string found in input
/Users/runner/work/rust/rust/tests/assembly/x86_64-array-pair-load-store-merge.rs:14:16: error: CHECK-NOT: excluded string found in input
 // CHECK-NOT: mov
               ^
/Users/runner/work/rust/rust/build/x86_64-apple-darwin/test/assembly/x86_64-array-pair-load-store-merge/x86_64-array-pair-load-store-merge.s:11:2: note: found here
 mov rbp, rsp

Input file: /Users/runner/work/rust/rust/build/x86_64-apple-darwin/test/assembly/x86_64-array-pair-load-store-merge/x86_64-array-pair-load-store-merge.s
Check file: /Users/runner/work/rust/rust/tests/assembly/x86_64-array-pair-load-store-merge.rs


-dump-input=help explains the following input dump.
Input was:
<<<<<<
<<<<<<
        1:  .section __TEXT,__text,regular,pure_instructions 
        2:  .macosx_version_min 10, 12 
        3:  .intel_syntax noprefix 
        4:  .globl _array_copy_2_elements 
        5:  .p2align 4, 0x90 
        6: _array_copy_2_elements: 
        7:  .cfi_startproc 
        8:  push rbp 
        9:  .cfi_def_cfa_offset 16 
       10:  .cfi_offset rbp, -16 
       11:  mov rbp, rsp 
not:14      !~~           error: no match expected
       12:  .cfi_def_cfa_register rbp 
       13:  movzx eax, word ptr [rdi] 
       14:  mov word ptr [rsi], ax 
       15:  pop rbp 
       16:  ret 
       17:  .cfi_endproc 
       19: .subsections_via_symbols 
>>>>>>
------------------------------------------

```

</details>

--- a/tests/assembly/x86_64-array-pair-load-store-merge.rs
+++ b/tests/assembly/x86_64-array-pair-load-store-merge.rs
@@ -2,6 +2,7 @@
 // compile-flags: --crate-type=lib -O -C llvm-args=-x86-asm-syntax=intel
 // only-x86_64
 // ignore-sgx
+// ignore-macos (manipulates rsp too)
 
 // Depending on various codegen choices, this might end up copying
 // a `<2 x i8>`, an `i16`, or two `i8`s.

